vaadin-spring-tests/test-ts-services/src/test/java/com/vaadin/flow/connect/AppViewIT.java

```diff
@@ -29,6 +30,7 @@
 /**
  * Class for testing issues in a spring-boot container.
  */
+@Ignore
```

Comment:
Could you add a comment for the ignore naming ticket 9751

Reply:
Done